sails.io.js/sails.io.js-tests.ts

```diff
@@ -0,0 +1,89 @@
+/// <reference path="./sails.io.js.d.ts"/>
+import * as SocketIO from 'socket.io-client';
```

Comment:
since this is an import of an `export =` module, the syntax should be `import SailsIOJS = require('sails.io.js');` to indicate that it doesn't work when emitting ES6-standard module code.